types/test-listen/test-listen-tests.ts

```diff
@@ -0,0 +1,25 @@
+import * as http from 'http';
+import * as testListen from 'test-listen';
```

Comment:
Since the package exports a function, that is not allowed in ES module, it should be `import testListen = require('test-listen')` here https://github.com/DefinitelyTyped/DefinitelyTyped#should-i-add-an-empty-namespace-to-a-package-that-doesnt-export-a-module-to-use-es6-style-imports

Reply:
OK, I'll change this. IMO `import X = require('x')` is "ugly", but I do understand that it's important to follow semantics.